source/PlayerInfoPanel.cpp

```diff
@@ -460,7 +460,10 @@ bool PlayerInfoPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comman
 	else if(command.Has(Command::MAP) || key == 'm')
 		GetUI()->Push(new MissionPanel(player));
 	else if(key == 'l' && player.HasLogs())
+	{
 		GetUI()->Push(new LogbookPanel(player));
+		return true;
+	}
```

Comment:
unrelated?
```suggestion
		GetUI()->Push(new LogbookPanel(player));
```

Reply:
I'll check, but it was probably done to prevent sound playing later on.